jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -99,14 +99,14 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
 
 
 @pytest.mark.filterwarnings('ignore:No observer defined on WCS')
-def test_write_momentmap(cubeviz_helper, spectrum1d_cube):
+def test_write_momentmap(cubeviz_helper, spectrum1d_cube, tmp_path):
     ''' Test writing a moment map out to a FITS file on disk '''
 
     # Simulate an existing file on disk to check for overwrite warning
-    test_file = Path("test_file.fits")
+    test_file = tmp_path / "test_file.fits"
```

Comment:
Generate test file should be jailed within the test directory. Then we should not worry about cleaning it up too.

Using `tmp_path` also comes with some convenience methods that simplify testing code, as follow.

Reply:
So the reason why this was done is because the moment map file writes it out to the current working directory. Unless we can run the tests from `tmp_path` maybe something like through `os.chdir()`, I don't think this won't trigger the overwrite protection?

Reply:
To clarify the above through example, the user could actually supply "../file.fits" to ask the plugin to write in the directory above it

Reply:
@duytnguyendtn , if you give it a path, it would write to the given path, so I think this test code is okay.

![Screenshot 2022-09-29 160618](https://user-images.githubusercontent.com/2090236/193131385-97b2965a-d1c6-4be9-8958-ce2a8622624f.jpg)